cohort_4/playbook/ACTIVITIES.md

```diff
@@ -1,6 +1,7 @@
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-generate-toc again -->
 **Table of Contents**
 
+- [Creating a safe space](#creating-a-safe-space)
```

Comment:
Let's not bother linking it in the README. I think it may be prematurely optimized because this section won't be referenced for the rest of Hack Camp. Also, I put it at the top because I thought that creating a safe space was one of our most important tenants.